src/assets/content.tsx

```diff
@@ -94,3 +94,90 @@ export const EMISSIONS_STAT_TITLE = 'Carbon Emissions Captured';
 // Other
 export const CITY_PLANTING_REQUEST_LINK =
   'https://www.cityofboston.gov/mayor/24?topic=new-tree-request';
+
+export enum Neighborhoods {
+  BACK_BAY = 'Back Bay',
+  CHARLESTOWN = 'Charlestown',
+  DORCHESTER = 'Dorchester',
+  DOWNTOWN = 'Downtown',
+  EAST_BOSTON = 'East Boston',
+  HYDE_PARK = 'Hyde Park',
+  JAMAICA_PLAIN = 'Jamaica Plain',
+  MATTAPAN = 'Mattapan',
+  MISSION_HILL = 'Mission Hill',
+  NORTH_END = 'North End',
+  ROSLINDALE = 'Roslindale',
+  ROXBURY = 'Roxbury',
+  SOUTH_BOSTON = 'South Boston',
+  WEST_ROXBURY = 'West Roxbury',
+  HARBOR_ISLANDS = 'Harbor Islands',
+  ALLSTON = 'Allston',
+  BRIGHTON = 'Brighton',
+  CHINATOWN = 'Chinatown',
+  LEATHER_DISTRICT = 'Leather District',
+  LONGWOOD = 'Longwood',
+  SOUTH_BOSTON_WATERFRONT = 'South Boston Waterfront',
+  BEACON_HILL = 'Beacon Hill',
+  WEST_END = 'West End',
+  SOUTH_END = 'South End',
+  BAY_VILLAGE = 'Bay Village',
+  FENWAY = 'Fenway',
+}
+
+export const NEIGHBORHOOD_IDS: { [id: number]: string } = {
+  2: Neighborhoods.BACK_BAY,
+  4: Neighborhoods.CHARLESTOWN,
+  6: Neighborhoods.DORCHESTER,
+  7: Neighborhoods.DOWNTOWN,
+  8: Neighborhoods.EAST_BOSTON,
+  10: Neighborhoods.HYDE_PARK,
+  11: Neighborhoods.JAMAICA_PLAIN,
+  12: Neighborhoods.MATTAPAN,
+  13: Neighborhoods.MISSION_HILL,
+  14: Neighborhoods.NORTH_END,
+  15: Neighborhoods.ROSLINDALE,
+  16: Neighborhoods.ROXBURY,
+  17: Neighborhoods.SOUTH_BOSTON,
+  19: Neighborhoods.WEST_ROXBURY,
+  22: Neighborhoods.HARBOR_ISLANDS,
+  24: Neighborhoods.ALLSTON,
+  25: Neighborhoods.BRIGHTON,
+  26: Neighborhoods.CHINATOWN,
+  27: Neighborhoods.LEATHER_DISTRICT,
+  28: Neighborhoods.LONGWOOD,
+  29: Neighborhoods.SOUTH_BOSTON_WATERFRONT,
+  30: Neighborhoods.BEACON_HILL,
+  31: Neighborhoods.WEST_END,
+  32: Neighborhoods.SOUTH_END,
+  33: Neighborhoods.BAY_VILLAGE,
+  34: Neighborhoods.FENWAY,
+};
+
+export const NEIGHBORHOOD_OPTS = [
+  { label: 'Back Bay', value: 2 },
+  { label: 'Charlestown', value: 4 },
+  { label: 'Dorchester', value: 6 },
+  { label: 'Downtown', value: 7 },
+  { label: 'East Boston', value: 8 },
+  { label: 'Hyde Park', value: 10 },
+  { label: 'Jamaica Plain', value: 11 },
+  { label: 'Mattapan', value: 12 },
+  { label: 'Mission Hill', value: 13 },
+  { label: 'North End', value: 14 },
+  { label: 'Roslindale', value: 15 },
+  { label: 'Roxbury', value: 16 },
+  { label: 'South Boston', value: 17 },
+  { label: 'West Roxbury', value: 19 },
+  { label: 'Harbor Islands', value: 22 },
+  { label: 'Allston', value: 24 },
+  { label: 'Brighton', value: 25 },
+  { label: 'Chinatown', value: 26 },
+  { label: 'Leather District', value: 27 },
+  { label: 'Longwood', value: 28 },
+  { label: 'South Boston Waterfront', value: 29 },
+  { label: 'Beacon Hill', value: 30 },
+  { label: 'West End', value: 31 },
+  { label: 'South End', value: 32 },
+  { label: 'Bay Village', value: 33 },
+  { label: 'Fenway', value: 34 },
+];
```

Comment:
Painfully redundant but quicker than going through the label/value pairs when trying to translate an id to a name - any other suggestions?

Reply:
Maybe a getNeighborhoodNames route that we can save in store? I think IDs/names are translated too often to rely on a getNeighborhoodName route. We can keep these for now and update in a future ticket - lmk what kind of backend route you think is best.